topologies/kne/cisco.config

```diff
@@ -0,0 +1,21 @@
+!! IOS XR Configuration 7.4.1
+!! Last configuration change at Wed Aug 18 19:55:09 2021 by cisco
+!
+username cisco
+ group root-lr
+ group cisco-support
+ password 7 01100F175804575D72
+! 
+grpc
+ dscp cs4
+ port 57400
+ max-streams 128
+ max-streams-per-user 128
+ address-family dual
+ max-request-total 256
+ max-request-per-user 32
+!
+ssh server vrf default
+end
+
```

Comment:
some extraneous blank lines can be deleted

Reply:
Done